data/multiplayer/scenarios/4p_Isars_Cross.cfg

```diff
@@ -59,4 +59,116 @@
         gold=75
         fog=yes
     [/side]
+
+    [options]
+        [checkbox]
+            id=classical_graphics
+            default=no
+            # po: This string will not only be used for Isar’s Cross.
```

Comment:
This hint doesn't say much about what the string to translate means. Does "classical" mean that the map will have orchestral music, the turn-timer replaced by an image of a conductor who waves their white baton as time runs out, and on the seventh turn the start-of-turn notification sound replaced with an unexpected cannon blast?

Reply:
Suggest `Simple background` for a name.

Reply:
I think the `name` should be written with title case (capital letters at the start of words), as title case is used for the "General" and "Game Settings" tabs.

Re-reading my previous comment, the joke about "classical" doesn't seem funny any more. Sorry for posting it.

Instead of a `#po` hint that it will be used for other maps, I'd suggest making it specific to this map for now.
```
    name=_ "isars^Disable Background Image"
    # po: Tooltip for the Isar's Cross map, makes the map look as it did in Wesnoth 1.14 and earlier
    description=_ "Show a blank area outside the map, instead of the floating island with cliffs."
```

Reply:
Classical Graphics was my attempt to positively paraphrase "Don't use fancy graphics", but it's sounds a lot better do be direct about what it does.